handout/advanced-components/access_child_components.md

```diff
@@ -1,15 +1,14 @@
  # Accessing Child Component Classes #
 
- ## @ViewChild & @ViewChildren ##
+ ## @ViewChild and @ViewChildren ##
+The @ViewChild and @ViewChildren decorators provide access to the classe of child component from the containing component.
 
-The `@ViewChild` & `@ViewChildren` decorators allow access to the classes of child components of the current component.
+The @ViewChild is a decarator function that takes the name of a component class as its input and finds it's selector in the template of the containing component to bind to. @ViewChild can also be passed a template reference variable.
 
-`@ViewChild` selects one class instance of the given child component class when given a type.
-
-For example, we can call `show` which is on the child component `Alert` class:
+For example, we bind the component class 'Alert' to its selector '<my-alert>' and assign it to the property 'alert'. This allows us to gain access to the functions in that class like Alert.show().
```

Comment:
Should use backticks instead of apostrophes for code snippets so that they get formatted properly